sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -580,6 +533,11 @@ def run_pipeline(self, pipeline, options):
       debug_options.add_experiment(
           'min_cpu_platform=' + worker_options.min_cpu_platform)
 
+    if pipeline.contains_external_transforms:
+      # All Dataflow multi-language pipelines use portable job submission by
+      # default.
+      debug_options.add_experiment("use_portable_job_submission")
```

Comment:
Just to check, this implies runner v2, right?

Reply:
Yes, updated to also include a Runner v2 check.